distribution/carbon-home/release-notes.html

```diff
@@ -42,9 +42,9 @@
 <ul>
     <li>Java 8 Support</li>
     <li><a href="https://docs.wso2.com/display/Carbon520/Architecture#Architecture-ComponentStartupOrderResolver">Carbon Component Startup Order Resolver</a></li>
-    <li>Carbon Tools - <a href="https://docs.wso2.com/display/Carbon520/Using+the+JAR+to+OSGi+Bundle+Converter">Jar To Bundle Converter</a> and <a href="https://docs.wso2.com/display/Carbon520/Dropins+Support+for+OSGi+Bundles#DropinsSupportforOSGiBundles-Updatethebundles.infofilepriortoserverlaunch">Dropins Installer</a></li>
+    <li>Carbon Tools - <a href="https://docs.wso2.com/display/Carbon520/Using+the+JAR+to+OSGi+Bundle+Converter">Jar To Bundle Converter</a> and <a href="https://docs.wso2.com/display/Carbon520/OSGiLib+Support+for+OSGi+Bundles#OSGiLibSupportforOSGiBundles-Updatethebundles.infofilepriortoserverlaunch">OSGi-Lib Installer</a></li>
     <li>Equipped with Eclipse Luna SR2 OSGi Framework</li>
-    <li><a href="https://docs.wso2.com/display/Carbon520/Dropins+Support+for+OSGi+Bundles#DropinsSupportforOSGiBundles-Updatethebundles.infofileduringserverlaunch">Dropins Support for OSGi Ready Bundles</a></li>
+    <li><a href="https://docs.wso2.com/display/Carbon520/OSGiLib+Support+for+OSGi+Bundles#OSGiLibSupportforOSGiBundles-Updatethebundles.infofileduringserverlaunch">OSGi-Lib Support for OSGi Ready Bundles</a></li>
```

Comment:
Reverted the doc url changes in commit 4571caa21afc93ee46f01727bfd8d4d418329f8f. Will update the urls once the doc is moved to git.

Thanks.